mp2-v1/src/values_extraction/mod.rs

```diff
@@ -161,6 +158,40 @@ pub fn identifier_for_mapping_value_column(
     compute_id_with_prefix(VALUE_ID_PREFIX, slot, contract_address, chain_id, extra)
 }
 
+/// Compute outer value indetifier for mapping of mappings variable.
+/// `outer_value_id = H(OUT_VAL || slot || contract_address || chain_id)[0]`
+pub fn identifier_for_outer_mapping_value_column(
+    slot: u8,
+    contract_address: &Address,
+    chain_id: u64,
+    extra: Vec<u8>,
+) -> u64 {
+    compute_id_with_prefix(
+        OUTER_VALUE_ID_PREFIX,
+        slot,
+        contract_address,
+        chain_id,
+        extra,
+    )
+}
+
+/// Compute inner value indetifier for mapping of mappings variable.
+/// `inner_value_id = H(IN_VAL || slot || contract_address || chain_id)[0]`
+pub fn identifier_for_inner_mapping_value_column(
```

Comment:
Already discussed offline, but to be sure we agree on this: these 2 methods shouldn't be necessary